### PR TITLE
Docs: Addon-A11y -  Documentation link to the docs for the test runner

### PR DESCRIPTION
**Pull request by Flambe**
## What I did

We've had some issues where we were trying to skip some tests and didn't realise that the documentation for this addon is split up in multiple places. The readme does not currently tell you that the accessibility rules aren't checked at all when using the test runner.

I've added a link in the readme to the documentation describing how to integrate the story rules with the test runner instead of copying the instructions as it duplicating it all just creates more places that need updating if anything changes.

#### The changes in this PR are covered in the following automated tests:
- [ ] stories
- [ ] unit tests
- [ ] integration tests
- [ ] end-to-end tests

#### Manual testing

It's just a readme change, can't really test that afaik

### Documentation

- [X] Add or update documentation reflecting your changes
- [ ] If you are deprecating/removing a feature, make sure to update
      [MIGRATION.MD](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md)

## Checklist for Maintainers

- [ ] When this PR is ready for testing, make sure to add `ci:normal`, `ci:merged` or `ci:daily` GH label to it to run a specific set of sandboxes. The particular set of sandboxes can be found in `code/lib/cli/src/sandbox-templates.ts`
- [ ] Make sure this PR contains **one** of the labels below:
   <details>
     <summary>Available labels</summary>

     - `bug`: Internal changes that fixes incorrect behavior.
     - `maintenance`: User-facing maintenance tasks.
     - `dependencies`: Upgrading (sometimes downgrading) dependencies.
     - `build`: Internal-facing build tooling & test updates. Will not show up in release changelog.
     - `cleanup`: Minor cleanup style change. Will not show up in release changelog.
     - `documentation`: Documentation **only** changes. Will not show up in release changelog.
     - `feature request`: Introducing a new feature.
     - `BREAKING CHANGE`: Changes that break compatibility in some way with current major version.
     - `other`: Changes that don't fit in the above categories.
   
   </details>

### 🦋 Canary release

This PR does not have a canary release associated. You can request a canary release of this pull request by mentioning the `@storybookjs/core` team here.

_core team members can create a canary release [here](https://github.com/storybookjs/storybook/actions/workflows/canary-release-pr.yml) or locally with `gh workflow run --repo storybookjs/storybook canary-release-pr.yml --field pr=<PR_NUMBER>`_

### CODE REVIEW

**Review by jonniebigodes — APPROVED** [review]
@Flambe, thanks for putting together this pull request and helping us improve the documentation. We appreciate it 🙏 ! So that you're aware, for this and other addons, the best documentation source is the official documentation and not the repo's Readmes, as they are a bit outdated and will be updated in the future to remove duplication,  avoid the maintenance burden and keep a single source of truth. Until then, I am more than glad to merge this once the checklist clears and help you and other Storybook users get the required information they need.

Hope you have a fantastic holiday season.

Stay safe

**Comment by Flambe** [comment]
Thank you for the info! I'll make sure the rest of the team is aware to use the main docs.

Have a great holiday as well!

**Pull request merged**
